.github/actions/npm-blacklist/action.yml

```
@@ -0,0 +1,42 @@
+    default: ''
+  additional-blacklist-pkgs:
+    description: 'List of additional packages (separated by YAML multiline), example: pkg@version'
+    required: false
+    default: ''
+  override-blacklist:
+    description: 'Path to a file with list of packages in form of "pkg@version" to override the default known blacklist'
+    required: false
+
+
+runs:
+    using: 'composite'
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 2 but found 4

(indentation)